### ISSUE TITLE
Sockets (device type 'cz') cannot be controlled after the new update

### ISSUE THREAD

**Issue by kirit1193** [tuya/tuya-homebridge#113]
Earlier all sockets of type 'cz' were controllable using this plugin, However, this is currently responding with a "command or value not support" error when I try to control it:

```
TuyaOpenAPI request: method = post, endpoint = https://openapi.tuyain.com, path = /v1.0/devices/5258562124a1600cfd26/commands, params = null, body = {"commands":[{"code":"10A Smart Plug","value":true}]}, headers = {"t":"1632504205132","client_id":"<snip>","nonce":"003730e1-1d57-11ec-a963-61ab65b6bc41","Signature-Headers":"client_id","sign":"7869E44228ADEC4FFA94525DF2540BA774B5F642E66CA81AF9C36E8A682BA2C7","sign_method":"HMAC-SHA256","access_token":"<snip>","lang":"en","dev_lang":"javascript","dev_channel":"homebridge","devVersion":"1.4.1"}

TuyaOpenAPI response: {"code":2008,"msg":"command or value not support","success":false,"t":1632504205230} path = /v1.0/devices/5258562124a1600cfd26/commands
```

**Comment by arpadarpa** [comment]
Same issue here. The problem is that the socket expects the following:
[{"code":"switch","value":true}]} 
instead of what you have [{"code":"DEVICE NAME","value":true}]}

Because the scheme of my devices is the following:

switch Boolean	"{true,false}"

countdown_1 Integer	
{
  "min": 0,
  "scale": 0,
  "unit": "s",
  "max": 86400,
  "step": 1
}

**Comment by bgphoenix** [comment]
Same issue here.

**Comment by Hanh94** [comment]
Hi guys, sorry for the late reply.
We have fixed it in 1.4.2, please upgrade and try again.

**Comment by arpadarpa** [comment]
> Hi guys, sorry for the late reply. We have fixed it in 1.4.2, please upgrade and try again.

Just checked. Issue has been resolved.  Thank you so much.

**Comment by kirit1193** [comment]
Issue has been resolved, closing this ticket. Thanks for the quick resolution!
